### ISSUE TITLE
install.bat fails to invoke config-distros.sh

### ISSUE THREAD

**Issue by spasche** [mintty/wsltty#232]
I'm running version 3.1.4 installed from Chocolatey.

I noticed that the "WSL Generate Shortcuts" script doesn't work properly. When running `install.bat` from CMD, it fails when trying to run the `config-distros.sh` script:
```
bin/dash: 0: Can't open C:UsersspascheAppDataLocalwsltty\config-distros.sh
```
It seems to be a regression from 2aace74c9da47c94d8e0891710c1d6bc7149e4bd. If I remove the quotes on `set installdir=%LOCALAPPDATA%\wsltty`, the `config-distros.sh` script is able to run from `install.bat`.

**Comment by mintty** [comment]
Yes, see also #230. I didn't revert the broken fix yet because there are also other problems with spaces in the user name.

**Comment by mintty** [comment]
Released installer 3.1.4.2, please check.
